src/helper.ts

```diff
@@ -64,7 +64,7 @@ export function mergeConfigs(
 }
 
 export function generateConfigs(configs: ConfigsOptions = {}): string {
-  const snakeCaseConfigs: { [key: string]: string | AuthAction | boolean | undefined } = {};
+  const snakeCaseConfigs: { [key: string]: string | AuthAction | boolean | AuthnMethod[] | undefined } = {};
```

Comment:
What about non array version?

Reply:
non-array version is covered by string. Typescript automatically recognizes that AuthnMethod is ultimately a string, but with fixed options